src/helpers/mapTreeToSuite.ts

```diff
@@ -39,6 +41,57 @@ const mapWorkspaceRootToSuite = (rootNode: WorkspaceRootNode): TestSuiteInfo | u
   };
 };
 
+const flatMapWorkspaceRootToSuite = ({ projects, id, label }: WorkspaceRootNode): TestSuiteInfo | undefined => {
+  if (!projects.length) {
+    return undefined;
+  }
+
+  const suite = projects.reduce<TestSuiteInfo>((results, project) => {
+    // Get an array of only the describe blocks in the current project.
+    const describeBlocks = findDescribeBlocksInNode(project);
+
+    // Map the describe blocks to a tree of suites, nested suites, and tests.
+    const currentChildren = describeBlocks.map(mapDescribeBlockToTestSuite);
+
+    // Merge this project's suites into the results.
+    const children = results.children.concat(currentChildren);
+
+    return { ...results, children };
+  }, {
+    children: [],
+    id,
+    label,
+    type: "suite",
+  });
+
+  // If the final children array is empty, return undefined to prevent the empty project from appearing in the list.
+  if (!suite.children.length) {
```

Comment:
Instead of casting the length to a truthy/falsy value using `!`, it might be better to check for an empty array explicitly. I understand this is a shortcut in JS but it looks like explicit checks are more favoured in this repo ([example 1](https://github.com/kavod-io/vscode-jest-test-adapter/blob/ccea0698c107912c243668ade4bcf5ffd90fc3d0/src/helpers/mapTestIdsToTestFilter.ts#L7), [example 2](https://github.com/kavod-io/vscode-jest-test-adapter/blob/7cb77e40fb64077ad9f9f49b1c36ed6a33f91400/src/helpers/mapJestTestResultsToTestEvents.ts#L18)) and they are easier to read.

```suggestion
  if (suite.children.length === 0) {
```

Reply:
No worries, totally reasonable. Updated!